samples/Azure.Management.Storage/Generated/Extensions/ArmClientExtensions.cs

```diff
@@ -13,193 +13,193 @@ namespace Azure.Management.Storage
     /// <summary> A class to add extension methods to ArmClient. </summary>
     public static partial class ArmClientExtensions
     {
-        #region BlobService
-        /// <summary> Gets an object representing a BlobService along with the instance operations that can be performed on it but with no data. </summary>
+        #region BlobServiceResource
+        /// <summary> Gets an object representing a BlobServiceResource along with the instance operations that can be performed on it but with no data. </summary>
         /// <param name="client"> The <see cref="ArmClient" /> instance the method will execute against. </param>
         /// <param name="id"> The resource ID of the resource to get. </param>
-        /// <returns> Returns a <see cref="BlobService" /> object. </returns>
-        public static BlobService GetBlobService(this ArmClient client, ResourceIdentifier id)
+        /// <returns> Returns a <see cref="BlobServiceResource" /> object. </returns>
+        public static BlobServiceResource GetBlobServiceResource(this ArmClient client, ResourceIdentifier id)
```

Comment:
This method name needs discussion as well, but this makes more sense with this name comparing with the occurrence that returns a `Collection`, after all this is returning a `BlobServiceResource` instance.